### ISSUE TITLE
How to indicate order of assays

### ISSUE THREAD

**Issue by kMutagene** [nfdi4plants/ARC-specification#60]
Given that one can chain assays by using the output of one assay as the input of another, are there any guidelines how to indicate that order?

**Comment by Brilator** [comment]
Isn't that implicitly done by using a sample (output) from one study / assay as the source (input) to another? 
By that you should be able to follow the complete "process sequence" (as @HLWeil called it yesterday) for an individual sample.

**Comment by kMutagene** [comment]
I am more focused on the experience when just browsing the ARC file structure. I have no way getting the process sequence other than opening the assay xlsx files and looking at sample names.

**Comment by Brilator** [comment]
Ah, yes. So a visualization of that ps. Didn‘t someone also work on a graph for that?

**Comment by kMutagene** [comment]
Not necessary visualization, a kind of readme or something would suffice. Maybe it would be best practice to write this file manually as the creator of the arc?

**Comment by Brilator** [comment]
This would double the annotation work and make it error-prone. I‘d rather teach good isa routines, have a tool read and visualize from isa and use that as „quality check“.

**Comment by Brilator** [comment]
It‘s the same as trying to manually visualize a workflow with its ins and outs vs. having seomthing read a CWL or other worfklow.

**Comment by kMutagene** [comment]
Okay let me rephrase. Ideally a tool will do this, yes. The tool is not there though. What is the suggested best practice until the tool is there?

**Comment by HLWeil** [comment]
Hey,
@Freymaurer and me built a visualization of the process sequence some time ago in `SWATE`, but it's still pretty much in the gimmick state .Also it only visualizes the connection of the processes of one assay, not between assays. 
To my knowledge, there is no current plan to add such a file to the `ARC`. If you want to have such a description, I think you can just do it like you want.

**Comment by kMutagene** [comment]
> Besides, it only visualizes the connection of the processes of one assay, not between assays

Which means that this issue is something that needs attention as it is not very user friendly to browse input and output columns of all assays to infer the assay order

**Comment by kMutagene** [comment]
This is more of a tooling issues, as the specification states that ISA is implemented, which means that the DAG is at least present implicitly. Closed as out-of-scope for the specs
